webapp/settings.py

```diff
@@ -2,6 +2,9 @@
 Django project settings
 """
 
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
```

Comment:
That's fair. But since you touched the line, it's now included in this diff, and now I'm questioning its existence. I don't think we should attach ourselves to what's standard in Django, as I have already removed a very large amount of the standard settings that come with Django. I think we should have as simple a settings file as possible, and to me that means removing this line. As its existence implies an app which knows what path at's at within the operating system, which somewhat breaks its encapsulation. If we can have an app which works without knowledge of its path, that would be better.

However, if you don't want to remove it now, I can submit a new PR after yours, and we can hold further discussion there. Up to you.